### DISCUSSION TITLE
Lost a limb?

### DISCUSSION THREAD

**Discussion by T-rexsuper1** [unitystation/unitystation#7073] [Ideas]
# Bioprinter
Needs to be researched. It uses special mushrooms for printing, which have be updated from botany, and genetically mutated to be suitable for humans.

# Augmentations
Mechanucal printer is aviable from the start. You can make simple limbs and organs from basic materials, but they arent that advanced. You can research more of them, or even some augmentations like skin shield, strength boosters, night vision, pain block implants and so on. Advanced stuff must be researched and uses advanced materials to be produced.

**Comment by Evan-R-Rogers** [comment]
This sounds more or less like a job for genetics than Botany, if I were to be honest.

**Comment by T-rexsuper1** [comment]
Okay than, but do you like the idea?

**Comment by ReiderJack** [comment]
> This sounds more or less like a job for genetics than Botany, if I were to be honest.

Might be interesting to connect those departments with this. Botanists to do (x) and genetics need to do (y) so they get something coool (z) from teaming up. Those interactions should be for all departments. So to get the best late game stuff a lot of departments need to interact to achive something. There are already interactions like that but they are not so deep.

**Comment by T-rexsuper1** [comment]
Yayyy

**Comment by SioxGWolf** [comment]
Welp, bioprinter could require biomass, which requires meat, the same for cloning.
Researching a bioprinter, I am fine with, and allowing mechanical limbs and cools mechanical abilities is just fun.
biomass is just chems and meat? Which requires botany or other departments to help.
That's typically what I am used to working within the medical field.
And if cloning is easy, why wouldn't I just print up a fresh clean body in the first place, and heck, use it to transplant limbs onto the original if they want to stay original.
Bioprinter and cloning need to have the same material cost and price if your gonna have it in at all.
Just gonna have this over from discord.
Ya, of course, need a DNA sample of the original to make the new limbs and organs, of course.
